btree/btree.c

```diff
@@ -8953,7 +8925,7 @@ static void ut_basic_tree_oper_icp(void)
 	void                   *temp_node;
 	int                     rc;
 	struct m0_fid           fid        = M0_FID_TINIT('b', 0, 1);
-	uint32_t                rnode_sz   = 1024;
+	uint32_t                rnode_sz   = 4096;
```

Comment:
Please use m0_getpagesize() instead of hardcode 4096.

Reply:
Done.